packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -814,7 +832,8 @@ class MediaQueryData {
         && other.boldText == boldText
         && other.navigationMode == navigationMode
         && other.gestureSettings == gestureSettings
-        && listEquals(other.displayFeatures, displayFeatures);
+        && listEquals(other.displayFeatures, displayFeatures)
+        && other.supportsShowingSystemContextMenu == supportsShowingSystemContextMenu;
```

Comment:
i hope the meta programming effort will allow us to auto synthesize these equal and hash functions, like what swift struct can do

Reply:
Agreed! I think that's the hope with the `@Model` annotation if I understand correctly.